aea/crypto/ethereum.py

```diff
@@ -103,6 +103,16 @@ def sign_message(self, message: str) -> bytes:
         signature = self._account.sign_message(m_message)
         return signature
```

Comment:
I think that we can remove the sign_message from the ethereum.py. If we are trying to make the ethereum.py and fetchai.py similar.